src/components/WearablePreview/WearablePreview.stories.tsx

```diff
@@ -91,12 +97,12 @@ storiesOf('WearablePreview', module)
       />
     </div>
   ))
-  .add('Preview a token from ropsten', () => (
+  .add('Preview a token from mumbai', () => (
     <div className="WearablePreview-story-container">
       <WearablePreview
-        contractAddress="0x07e2a8fce1eb1ddf8e7d8de3f86654abe32eb97f"
+        contractAddress="0xf88951ddd9ff334ae98901e26042edbb8f6d52e5"
```

Comment:
I changed this address because when the testnet catalyst were migrated to goerli this collection was lost in the process, so I replaced it with one that works